frontend/src/components/Instruction.jsx

```diff
@@ -53,12 +53,7 @@ function Instruction() {
         </li>
       </ul>
       <div className="userButton">
-        <Link
-          to="/categories/hebergement"
-          type="button"
-          className="userBtn"
-          Hebergement
-        >
+        <Link to="/categories/1" type="button" className="userBtn" Hebergement>
```

Comment:
There is no attribute `type="button"` for a `<Link>` and attribute `Hebergement` doesn't too.